### ISSUE TITLE
Case sensitivity

### ISSUE THREAD

**Issue by mastercoms** [p0358/vdf-parser#8]
Hi, wonderful library! I've noticed that for the reference VDF implementation, keys are case-insensitive. However, keys in this library are case-sensitive. Could a parse option be added to make options case insensitive? Perhaps by just adding toLowerCase for all keys. This would suffice for my usage, at least.

**Comment by joekiller** [comment]
I agree this would be helpful. For reference, you can see that valve utilizes a [tolower function](https://github.com/ValveSoftware/source-sdk-2013/blob/0d8dceea4310fde5706b3ce1c70609d72a38efdf/sp/src/tier1/utlbuffer.cpp#L1174)
